pkgs/development/tools/misc/sipp/default.nix

```diff
@@ -1,29 +1,30 @@
-{lib, stdenv, fetchurl, ncurses, libpcap }:
+{ lib, stdenv, fetchurl, ncurses, libpcap, cmake, openssl, git }:
 
 stdenv.mkDerivation rec {
-  version = "3.6.0";
+  version = "3.6.1";
 
   pname = "sipp";
 
   src = fetchurl {
```

Comment:
Use `fetchFromGitHub` instead, it will do the right thing.

Reply:
I'm not sure as the release notes state that some necessary files aren't present within the github source:
>NOTE: Please download sipp-3.6.1.tar.gz from link below instead of Source code (tar.gz). They are both source code, but the latter one is a snapshot of the git v3.6.1 tag and lacks version.h and sipp.1.